src/storage/CookiesStorage.js

```diff
@@ -127,7 +127,17 @@ class CookiesStorage {
   }
 
   /** @override */
-  load(key, defaultValue) {}
+  load(key, defaultValue) {
+    if (!document.cookie.includes(`${key}=`)) {
+      return defaultValue;
+    }
+
+    const cookieValueJSON = document.cookie.split('; ')
+      .find((row) => row.startsWith(`${this.settings_['prefix']}${key}`))
```

Comment:
Per cookie syntax, each cookie (`row` here) *should*, although may not always, start with a space, and may contain additional whitespace around the cookie name.

https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Syntax

So, these are all valid and equivalent cookies that the code should handle:

```
"my_cookie=my_value"
" my_cookie=my_value"
"     my_cookie    = my_value "
```

Reply:
Instead of startswith, I changed it to indexOf(prefix + key) >= 0;

This recognizes the cookie even with whitespace, and JSON is not affected by it.
Does that seem ok?

Reply:
I think this will run into trouble if a cookie's name appears in the value of another cookie, or if two cookies have names where one is a subset of the other.

A few more example cookie strings:
```
"my_cookie=my_value; my_cookie_2=my_other_value"
"my_cookie=my_value; my_favorite_cookie=my_cookie"
// And the browser may have things ordered in reverse.
```

In either case, if I ask for the value of "my_cookie", I might get something unexpected back.